#! /bin/sh /usr/share/dpatch/dpatch-run
## 02_drraw_conf.dpatch by Mohammad Ebrahim Mohammadi Panah <ebrahim@mohammadi.ir>
##
## All lines beginning with `## DP:' are a description of the patch.
## DP: Patch for drraw.conf
## DP: Sets correct data and cache directories for the Debian package

@DPATCH@

--- drraw-2.1.3/drraw.conf	2004-11-13 23:31:22.000000000 +0330
+++ drraw-2.1.3.patched/drraw.conf	2008-12-05 20:02:59.000000000 +0330
@@ -69,8 +69,8 @@
 # These directories MUST EXIST and the user running CGI scripts must have
 # read AND write access to them.  It is entirely safe to delete any content
 # found in the temporary directory.
-$saved_dir = '/somewhere/drraw/saved';
-$tmp_dir = '/somewhere/drraw/tmp';
+$saved_dir = '/var/lib/drraw';
+$tmp_dir = '/var/cache/drraw';
 # By default, critical errors are shown in the produced HTML pages produced
 # and sent to standard error (which web servers typically write to some
 # logfile).  If the following is defined, such errors will be written to
